script/sendmail.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+#for saving the mail temporarily
+random=$RANDOM
+
+#Mail related functionality
+mail=$(which mailutils)
+mail_status=$?
+check_ser=$(which postfix)
+check_ser_status=$?
+temp_d=/tmp/
+mail_ext=.mail
+
+#Checking GUI related functions
+gui_check="$(startx >/dev/null 2>&1)"
+gui_status=$?
+
+
+echo -e "THIS PROGRAM IS OF NO WARRANTY!!! I AM NOT RESPONSIBLE FOR ANY 
+MISUSES OF IT. THIS SCRIPT IS SOLELY FOR EDUCATIONAL PURPOSES \n \n"
+sleep 5
+clear
+
+
+if [ $mail_status != 0 ]
+then
+	echo "I am very sorry... But I require the mailutils package to sendmail"
+	echo "I suggest using apt or yum to install"
+	exit 1
+fi
+
+if [ $check_ser_status != 0 ]
+then
+	echo "I am very sorry... But I require the Postfix service"
+	echo "I suggest using apt or yum to install"
+	exit 1
+fi
+
+if [ $gui_status == 0 ]
+then
+	echo "WARNING!!! Running on home Computer is highly not recommended due to the increased chance of being mislead as spam"
+	sleep 2
+	exit 1
+fi
+
+
+read -p "Enter from address: " from 
+read -p "Enter receiving address: " receive
+read -p "Enter subject: " subject
+echo "Enter message in one second: "
+sleep 2
+#Open a file to edit the body of a message
+nano $temp_d$random$mail_ext
+body=$(cat $temp_d$random$mail_ext)
+
+read -p "Ready to send? (yes): " send
+
+if [ $send == "yes" ]
+then
+	echo $body | mail -aFrom:$from -s "$subject" $receive
```

Comment:
`mail` on different systems may be different. e.g. `mailx` may skip postfix and use a remote smtp instead if the mailrc is configured so: https://www.systutorials.com/1411/sending-email-from-mailx-command-in-linux-using-gmails-smtp/ . The script seems try to use postfix. Is there a way/option to force so here?

Reply:
I believe that mailutils generally always uses postfix. At least in my experience anyways.

Reply:
"Normally, mailx invokes sendmail(8) directly to transfer messages. If the smtp variable is set, a SMTP connection to the server specified by the value of this variable is used instead. " from mailx manual https://www.systutorials.com/docs/linux/man/1-mailx/ . This should be the behavior of the mailx implementation.

Here, I think you want to force `mail`/`mailx` to call `sendmail`. Setting the `smtp` variable to empty may play the trick: `mail -S smtp="" <other arguments/options>`.

`mail` invokes `sendmail` which may be from postfix or other tools. The `sendmail` may be from another MTAs such as exim or ssmtp. So instead of checking `postfix`, checking `sendmail` may be better.

Reply:
Sounds like you know what you're doing, feel free to modify the code as you see fit.

Reply:
Sounds good. I will merge and then make some further changes to this script. Thanks for the pulling request.